forwarder/Forwarder.go

```diff
@@ -0,0 +1,53 @@
+package forwarder
+
+import (
+	"bytes"
+	"fmt"
+	tidUtil "github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Forwarder struct {
```

Comment:
I don't really know how go works? Can I use (not create, use) an instance of Forwarder if it is not exposed?